Topic/rdkemw 4206

anaswaragk opened the pull request.

Commit:
RDKEMW-4206 : subtec-app code rebase

Reason for change: Compare and rebase
Test Procedure: Regression test
Risks: Medium
Signed-off-by:Anaswara KookkalAnaswara_Kookkal@comcast.com

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/subtec-app?pullRequest=27) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/subtec-app?pullRequest=27) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: krithikasvraman<br/>:x: Anaswara G K<br/><hr/>**Anaswara G K** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/subtec-app?pullRequest=27) it.</sub>

krithikasvraman committed:
changes from RDKAAMP-1100 and RDKAAMP-77

Commit:
RDKEMW-4206 : subtec-app code rebase

Reason for change: Compare and rebase
Test Procedure: Regression test
Risks: Medium
Signed-off-by:Anaswara KookkalAnaswara_Kookkal@comcast.com

krithikasvraman committed:
RDKEMW-4206: Changes from commits afb011c and 341da1e

Reason for change: compare and rebase
Test Procedure: Regression test
Risks: Medium
Signed-off-by:krithika_venkataraman@comcast.com

krithikasvraman committed:
RDKEMW-4206: Changes from commits 81ade1a, 7801b73 and b4fb7c4

Reason for change: compare and rebase
Test Procedure: Regression test
Risks: Medium
Signed-off-by:krithika_venkataraman@comcast.com
